### ISSUE TITLE
"Documenting your API" topic slightly outdated

### ISSUE THREAD

**Issue by rpkilby** [encode/django-rest-framework#6987]
Was fixing a broken link and noticed that the referenced section no longer exists in the schemas docs. 

https://github.com/encode/django-rest-framework/blob/9850441e6f58d67f4d533ee144aeef54724ac661/docs/topics/documenting-your-api.md#L224-L225

**Comment by markopy** [comment]
The missing documentation is for the ViewSet docstring format. See: https://stackoverflow.com/questions/57367230/how-to-dinstinctly-document-possible-rest-actions-in-viewset-docstring/58435631#58435631

**Comment by cyberpunk-akash** [comment]
Hi, I'm new to open source and wish to contribute to this repo as I am an active user of Django-REST. Have sufficient knowledge of Git. Can someone please explain in layman terms how to clear this issue. Would be very grateful of you. Cheers!

**Comment by rpkilby** [comment]
Hi @cyberpunk-akash. Thanks for your interest! There isn't really a direct fix here - it's more a question of how we want to structure this specific docs section, which is a question for other maintainers. e.g., do we want to leave it in the outdated coreapi docs, or do we want to move it into the main docs? Are the specific examples still supported, or are they no longer relevant given that coreapi is no longer supported? 

Closing, as this issue is being addressed in #6967.
